auth/src/android/auth_android.cc

```diff
@@ -774,6 +775,18 @@ void Auth::SignOut() {
   SetImplFromLocalRef(env, nullptr, &auth_data_->user_impl);
 }
 
+void Auth::UseEmulator(const std::string host, uint32_t port) {
+  JNIEnv* env = Env(auth_data_);
+  jstring j_host = env->NewStringUTF(host.c_str());
+  env->CallVoidMethod(AuthImpl(auth_data_), auth::GetMethodId(auth::kUseEmulator), j_host, port);
+  env->DeleteLocalRef(j_host);
+  firebase::util::CheckAndClearJniExceptions(env);
+}
+
+std::string Auth::GetEmulatorUrl() {
```

Comment:
Shouldn't we either save the host/port and return a URL here, or get it from the Android SDK?

Reply:
Changed the way to use environment settings